src/listeners/commands/_chat-input-shared.ts

```diff
@@ -0,0 +1,18 @@
+import { getSupportedUserLanguageT } from '#lib/i18n/translate';
+import { type ChatInputCommandErrorPayload } from '@sapphire/framework';
+import type { ChatInputSubcommandErrorPayload } from '@sapphire/plugin-subcommands';
+import { MessageFlags } from 'discord.js';
+import { flattenError, generateUnexpectedErrorMessage, resolveError } from './_shared.js';
```

Comment:
```suggestion
import type { ChatInputSubcommandErrorPayload } from '@sapphire/plugin-subcommands';
import { flattenError, generateUnexpectedErrorMessage, resolveError } from './_shared.js';
```

If the other 2 are accepted